11ty/definitions/southeast-asian.md

```diff
@@ -2,6 +2,7 @@
 title: Southeast Asian
 slug: southeast-asian
 defined: true
+excerpt: umbrella term for people of Southeast Asian descent; the two main subregions are Mainland Southeast Asia and Maritime Southeast Asia
```

Comment:
I'm unsure whether we should include a full list of countries here. For East Asian and South Asian I included the list of countries; however, the list of countries for Southeast Asian is significantly longer. I compromised here by providing the two main subregions, but I'm not sure how many people would be familiar with the countries within.

Reply:
Paging @tatianamac for feedback on this :)

Reply:
@thejoycekung I LOVEEEEE that you're being mindful of this. In this instance since it's just for the excerpt, I think we're good.